README.md

```diff
@@ -1,24 +1,88 @@
-# README
+# Mentor-booking-App-Backend
 
-This README would normally document whatever steps are necessary to get the
-application up and running.
+This app is aim to connect mentor and mentee from around the world, to make software development journey easier for the newbies developer by getting tips, ideas and guide from experience software developer (mentor).
 
-Things you may want to cover:
+## Built With
 
-* Ruby version
+- Ruby
+- Ruby on Rails (v7.0.3.1)
+- postgresql
+- postman
 
-* System dependencies
+## Built With
 
-* Configuration
+<a href='https://www.ruby-lang.org/en/'><img src='https://external-content.duckduckgo.com/iu/?u=https%3A%2F%2F4.bp.blogspot.com%2F-lvBDcRp_jl8%2FVzRp2rLNttI%2FAAAAAAAAAYs%2F4YivgL8duHktR32feBMVY8kFqe2K2A4sQCLcB%2Fs1600%2Fruby-1212.png&f=1&nofb=1' height='60' alt='Ruby Logo' aria-label='https://www.ruby-lang.org/en/' /></a>
 
-* Database creation
+## Get Started
 
-* Database initialization
+```
+To get a clone of the project, run `git clone https://github.com/zmekonnen251/Mentor-Booking-App-Backend.git`
 
-* How to run the test suite
+```
 
-* Services (job queues, cache servers, search engines, etc.)
+```
+cd `Mentor-booking-App-Backend`
+```
 
-* Deployment instructions
+### Available Scripts
 
-* ...
+In the project directory, you can run:
+
+**`bundle install`** to install require tools for the project to run
+
+**`rails server`**
+
+**`http://localhost:3000/`**
+
+**`Rspec spec`** to run all the test
+
+For more details, see [Ruby on Rails Guides (v7.0.3.1)](https://guides.rubyonrails.org/).
+
+## Authors
+
+👤 **Adekunle Ismail**
+
+- GitHub: [@ismailakinkunmi](https://github.com/ismailakinkunmi)
+- LinkedIn: [Adekunle Ismail](https://www.linkedin.com/in/adismail4/)
+
+👤 **Zelalem Mekonnen**
+
+- GitHub: [@zmekonnen251](https://github.com/zmekonnen251)
+- LinkedIn: [Zelalem Getachew](https://www.linkedin.com/in/zelalem-getachew/)
+- Twitter: [Zelalem Getachew](https://twitter.com/mek_zela)
+
+👤 **Alfred Bisimwa**
+
+- GitHub: [@Alfredbis29](https://github.com/Alfredbis29)
+- Twitter: [Alfred Bisimwa](https://twitter.com/AlfredBisimwa1)
+- LinkedIn: [Alfred Bisimwa](https://www.linkedin.com/in/alfred-bisimwa-0501a81a8/)
+
+👤 **zieeco**
+
+- GitHub: [@zieeco](https://github.com/zieeco)
+- Twitter [@zieecodes](https://twitter.com/zieecodes)
+- LinkedIn: [Isaac Imaobong Samuel](https://www.linkedin.com/in/isaac-imaobong-samuel)
+
+👤 **Wuletaw Wonte**
+
+- GitHub: [@wuletawwonte](https://github.com/wuletawwonte)
+- Twitter: [Wuletaw Wonte](https://twitter.com/wuletaww)
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
+
+- Hat tip to microverse for their support
+- Inspiration
+
+## 📝 License
+
+This project is [MIT](./MIT.md) licensed.
```

Comment:
- He great job with the readme, however, there's an issue with the license link as it's broken, please, kindly update the MIT link to make your readme more professional.